commands/info_test.go

```diff
@@ -9,12 +9,52 @@ import (
 // Test_PrintInfo simply executes the printInfo function.
 // TODO: actually test what this does
 func Test_PrintInfo(t *testing.T) {
+	// our test config YAML
+	yamlText := `last_version_check: 0001-01-01T00:00:00Z
+updated: 2017-09-29T11:23:15+02:00
+endpoints:
+  https://myapi.domain.tld:
```

Comment:
Not related to this PR but just to have that said. IMO there should either be an explicit ID to identify an endpoint, or none at all. If there is an ID it should be the codename IMO. If there is none the data structure should be a list of objects. IMO the API endpoint of an installation is just one property of many an installation can have. IMO it is not correct to use the API endpoint as identifier.